addons/account/models/account.py

```diff
@@ -769,6 +769,7 @@ def create(self, vals):
                 company = self.env['res.company'].browse(company_id)
                 account_vals = self._prepare_liquidity_account(vals.get('name'), company, vals.get('currency_id'), vals.get('type'))
                 default_account = self.env['account.account'].create(account_vals)
+                default_account.onchange_code()
```

Comment:
With the CoA based on groups the onchange helps with this user story line:

**Given that I create a new account then trigger the correct group instead make the user/accountant to look for it**.

In Perú it is legally required the CoA, then almost all accountants simply **Know** the code and the more common behavior is create the account (by any mean) and call the onchange (that set the proper group).

But yes this can be discussed in a different PR just if we do this one then the the group_id:id entry on the csv can be removed for example (making consistent the behavior as if we create manually the accounts).

Reply:
This is being done in a [different PR](https://github.com/odoo/odoo/pull/33079), without this change then all accounts autocreated are not having the proper group once create (even if it is automated).

![image](https://user-images.githubusercontent.com/338634/57201388-fdab1480-6f5d-11e9-98e6-039e4157d88b.png)